RocketChat/Rocket.Chat.ReactNative
# [FIX] Omnichannel chat is broken the first time is taken

## Proposed changes

Omnichannel chat is broken the first time is taken

## Issue(s)	


## How to test or reproduce


- Be a livechat-agent
- Then in sequence take a room -> click on the header -> click on the info
- You should be redirected to RoomInfoView
- The infos in RoomInfoView should be as normal

## Screenshots

#### Before
https://user-images.githubusercontent.com/47038980/162280086-64cc4d84-9078-4efa-83d1-4d19c9a6b316.mov

#### After
https://user-images.githubusercontent.com/47038980/162279911-6e42ce4b-9853-4b81-9e38-58a4e31febc9.mov


## Types of changes




## Checklist



## Further comments
